src/index.html

```diff
@@ -17,6 +17,22 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href = "#" class="logo">
+        <img src="images/logo.png" alt="Moyo Logo">
+      </a>
+      <nav>
+        <ul>
+          <li><a href="#" class ="is-active">Apple</a></li>
+          <li><a href="#">Samsung</a></li>
+          <li><a href="#">Smartphones</a></li>
+          <li><a href="#" class ="hover">Laptops & Computers</a></li>
```

Comment:
The `.hover` class is not functioning as intended. Please check the CSS file to ensure that the class is defined correctly and that the styles are applied as expected.